src/AllLayers.js

```diff
@@ -61,6 +37,31 @@ L.LayerGroup.environmentalLayers = L.LayerGroup.extend(
 
       this.options.layers = param;
 
+      this._OpenInfraMap_Power = L.tileLayer('https://tiles-{s}.openinframap.org/power/{z}/{x}/{y}.png', {
```

Comment:
I am a bit worried about moving these layers here but this seems to fix the issue of cross-wiring between layers when we have multiple maps on the page as mentioned here https://github.com/publiclab/leaflet-environmental-layers/pull/443#issuecomment-595771022

It's hard to test how these layers behave when added to the map because they don't work anymore. We could also comment them out till it is fixed.

Reply:
Ah ok -- and noting #387 as the bug to fix for that. Thanks!